sonar-db/src/test/java/org/sonar/db/issue/IssueDaoTest.java

```diff
@@ -124,18 +129,40 @@ public void get_by_key() {
   }
 
   @Test
-  public void get_by_keys() {
+  public void selectByKeyOrFail_fails_if_key_not_found() {
```

Comment:
I'm used to prefix failing tests with fail_when in this case I would replace with fail_if_key_not_found_with_selectByKeyOrFail